api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java

```diff
@@ -1208,7 +1208,6 @@ public static CohortDefinition getMQ11DenB4(
             + "                     AND o.value_coded = ${1065}    "
             + "                     AND e.encounter_datetime BETWEEN :startDate AND :endDate   "
             + "                     AND e.location_id = :location    "
-            + "             GROUP BY p.patient_id  "
```

Comment:
There is a MIN function that is used in this cohort definition for grouping patients, if you remove this, it will mean that you will only have one patient for that matter. Is this the only change you made?

Reply:
Ignore this, seen where changes is effected